runtime/src/bank.rs

```diff
@@ -1385,6 +1385,7 @@ impl Bank {
     /// A snapshot bank should be purged of 0 lamport accounts which are not part of the hash
     /// calculation and could shield other real accounts.
     pub fn verify_snapshot_bank(&self) -> bool {
+        self.purge_zero_lamport_accounts();
```

Comment:
@aeyakovenko Err, I think this is removed because this is cherry-picked from https://github.com/solana-labs/solana/pull/7010, which is not a correct fix (I'm working on it), and isn't related to this PR.

Reply:
back ref: https://github.com/solana-labs/solana/pull/7010#discussion_r349843928